libs/core-ui/.eslintrc.json

```diff
@@ -11,6 +11,12 @@
       "rules": {
         "max-lines": "warn"
       }
+    },
+    {
+      "files": ["**/getBubbleChartOptions.ts"],
+      "rules": {
+        "dot-notation": "off"
```

Comment:
why did this need to be turned off for this change ?

Reply:
There is a code like this["xyz"] which will be converted to this.xyz from lintfix which will result in saying xyz is not in this. As 'xyz' is a custom param added to point in highchart and could not be identified onClick.